sqlglot/expressions.py

```diff
@@ -428,6 +428,69 @@ def assert_is(self, type_):
         assert isinstance(self, type_)
         return self
 
+    def union(self, expr, distinct=True, dialect=None, **opts):
+        """
+        Builds a UNION expression.
+
+        Example:
+            >>> import sqlglot
+            >>> sqlglot.parse_one("SELECT * FROM foo").union("SELECT * FROM bla").sql()
+            'SELECT * FROM foo UNION SELECT * FROM bla'
+
+        Args:
+            expr (str or Expression): the SQL code string.
+                If an `Expression` instance is passed, it will be used as-is.
+            distinct (bool): set the DISTINCT flag if and only if this is true.
+            dialect (str): the dialect used to parse the input expression.
+            opts (kwargs): other options to use to parse the input expressions.
+        Returns:
+            Union: the Union expression.
+        """
+        expr = maybe_parse(sql_or_expression=expr, dialect=dialect, **opts)
+        return Union(this=self, expression=expr, distinct=distinct)
```

Comment:
I think this can be `union(left=self, right=expr, distinct=distinct, dialect=dialect, **opts)`